apps/qubit/modules/informationobject/templates/_identifierOptions.php

```diff
@@ -0,0 +1,11 @@
+<div class="section">
```

Comment:
Yeah, I actually noticed this and was struggling on what to do. I figured maybe if we add the alt identifiers to the other standards, we could then add in alternative identifier functionality to them too in another ticket soon. But I can pass a boolean in as well to hide it for now as you suggest. Maybe there's a reason mods and dc don't have alternative identifiers? I'm not sure!